_layouts/default.html

```diff
@@ -2,11 +2,12 @@
 <html lang="en">
 {% include head.html %}
 <body>
-
+  
```

Comment:
finally the modification of the default hatm to be able to place the scripts in the correct structure